fiftyone/server/routes/camera.py

```
@@ -0,0 +1,166 @@
+        The dataset
+    """
+    try:
+        return fou.load_dataset(id=dataset_id)
+    except ValueError as err:
+        raise HTTPException(
+            status_code=404,
+            detail=f"Dataset '{dataset_id}' not found",
+        ) from err
+
+
+def _get_sample(dataset, sample_id: str):
```

Ruff (0.14.10)

Missing return type annotation for private function `_get_sample`

(ANN202)